src/main/java/com/alkemy/ong/service/CommentService.java

```diff
@@ -23,4 +25,7 @@ public CommentDto save(CommentDto dto) {
             return commentMapper.commentEntityToCommentDto(commentRepository.save(comment));
         }
 
+         public List<CommentBodyDto> getAll() {
```

Comment:
suggestion for the method name: getCommentsOrderByDate or getCommentsOrderByTimestamps, what do you think?

Reply:
Your suggestion is good, although for this time I would like to keep the name :-).